CompanionApp/Companion/Views/Onboarding/Templates/WalkthroughBaseStepItemTemplate.cs

```diff
@@ -185,13 +185,13 @@ public string AnswerSix
         public static readonly BindableProperty AnswerSixVisibleProperty =
             BindableProperty.Create(
                 nameof(AnswerSixVisible),
-                typeof(string),
+                typeof(bool),
                 typeof(WalkthroughBaseStepItemTemplate),
-                string.Empty);
+                false);
 
-        public string AnswerSixVisible
+        public bool AnswerSixVisible
         {
-            get { return (string)GetValue(AnswerSixVisibleProperty); }
+            get { return (bool)GetValue(AnswerSixVisibleProperty); }
             set { SetValue(AnswerSixVisibleProperty, value); }
         }
 
```

Comment:
It looks like you removed the need for these properties. Can they go ahead and be deleted?